runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -540,6 +552,35 @@ ClassFileOracle::walkAttributes()
 	}
 }
 
+void
+ClassFileOracle::checkHiddenClass()
+{
+	ROMClassVerbosePhase v(_context, ClassFileAttributesAnalysis);
+	/* Hidden Class cannot be a record or enum. */
+	U_16 superClassNameIndex = getSuperClassNameIndex();
+	bool isEnum = false;
+	if (0 != superClassNameIndex) {
+		isEnum = J9_ARE_ALL_BITS_SET(_classFile->accessFlags, CFR_ACC_ENUM) && 
+		J9UTF8_DATA_EQUALS(getUTF8Data(superClassNameIndex), getUTF8Length(superClassNameIndex), "java/lang/Enum", LITERAL_STRLEN("java/lang/Enum")) &&
```

Comment:
Some kind of indenting is called for here.

Reply:
I'd much prefer that the comments not appear inside the if condition.

Reply:
Fixed indenting and moved the comment outside of the if check.